.github/scripts/check-anchors.py

```
@@ -0,0 +1,248 @@
+    # normalize unicode so e.g. full-width chars compare sanely
+    text = unicodedata.normalize('NFKC', text)
+    out = []
+    for ch in text:
+        if ch.isalnum() or ch in '+-_':
+            out.append(ch)
+        elif ch in ' \t':
+            out.append(' ')
+        # everything else dropped (punctuation like : ( ) ? . , & / >)
+    slug = ''.join(out)
+    slug = re.sub(r'\s+', '-', slug).strip('-')
+    return slug
```

Ruff (0.16.1)

[warning] Unnecessary assignment to `slug` before `return` statement

Remove unnecessary assignment

(RET504)